data/pyridazine.yml

```diff
@@ -0,0 +1,32 @@
+# pyridazine.yml
+A: 6242.95041
+B: 5961.09410
+C: 3048.71363
+Delta_J: 0.75676E-3
+Delta_JK: -0.13558E-3
+Delta_K: 0.82976E-3
+del_J: 0.315715E-3
+del_K: 0.68234E-3
+Phi_J: 0.000233E-6
+Phi_JK: -0.00132E-6
+Phi_KJ: 0.00158E-6
+Phi_K: 0.0
+phi_J: 0.000190E-6
+phi_JK: -0.000252E-6
+phi_K: 0.002081E-6
+chi_aa_1: -4.6411
+chi_bb_1: 1.3711
+chi_cc_1: 3.2700
+chi_ab_1: 2.709
+chi_aa_2: -4.6411
+chi_bb_2: 1.3711
+chi_cc_2: 3.2700
+chi_ab_2: 2.709
```

Comment:
I think the off-diagonal chi_ab constants for 1 and 2 should have signs opposite of each other, but which one is + and which one is - is not determined.

Reply:
great - that sounds reasonable to me. i knew you would have the answer bryan!